src/css/_headers.scss

```diff
@@ -0,0 +1,41 @@
+header {
+  text-align: center;
+  height: 9vw;
+}
+
+button:active {
+  border: none;
+  outline: none;
+}
+
+#profile-button {
```

Comment:
I'm still not sure why we can't get the SCSS to work when we try to break it down into smaller subfiles. Maybe something to circle back on if we have the time.